superbench/benchmarks/micro_benchmarks/cudnn_function.py

```diff
@@ -0,0 +1,441 @@
+                'dilationA': [1, 1],
+                'mode': 1,
+                'use_tensor_core': True,
+                'input_type': 2,
+                'conv_type': 0,
+                'algo': 1
+            }
+        ]
+
+        self._bin_name = 'CudnnBenchmark'
+
+    def add_parser_arguments(self):
+        """Add the specified arguments."""
+        super().add_parser_arguments()
+        self._parser.add_argument(
+            '--num_warmup',
+            type=int,
+            default=8,
+            required=False,
+            help='The number of warmup step.',
+        )
+        self._parser.add_argument(
+            '--num_steps',
+            type=int,
+            default=100,
+            required=False,
+            help='The number of test step.',
+        )
+        self._parser.add_argument(
+            '--num_in_step',
+            type=int,
+            default=1000,
+            required=False,
+            help='The number of functions in one step.',
+        )
+        self._parser.add_argument(
+            '--random_seed',
+            type=int,
+            default=33931,
+            required=False,
+            help='The random seed to fill in the data of the function.',
+        )
+        self._parser.add_argument(
+            '--config_json_str',
+            type=str,
+            default=None,
+            required=False,
+            help='The custom json string defining the params in a cudnn function.',
+        )
+
+    def _preprocess(self):
+        """Preprocess/preparation operations before the benchmarking.
+
+        Return:
+            True if _preprocess() succeed.
+        """
+        if not super()._preprocess():
+            return False
+
+        command = os.path.join(self._args.bin_dir, self._bin_name)
+        command += (' --num_test ' + str(self._args.num_steps))
+        command += (' --warm_up ' + str(self._args.num_warmup))
+        command += (' --num_in_step ' + str(self._args.num_in_step))
+        command += (' --random_seed ' + str(self._args.random_seed))
+
+        try:
+            if not self._args.config_json_str:
+                for config_dict in self.__default_params_dict_list:
+                    config_json_str = "\'" + json.dumps(config_dict).replace(' ', '') + "\'"
+                    print(config_json_str)
+                    complete_command = command + (' --config_json ') + config_json_str
+                    self._commands.append(complete_command)
+
+            else:
+                custom_config_str = yaml.safe_load(self._args.config_json_str)
+                config_json_str = "\'" + json.dumps(custom_config_str).replace(' ', '') + "\'"
+                complete_command = command + (' --config_json ') + config_json_str
+                self._commands.append(complete_command)
+        except BaseException as e:
+            logger.error('Invalid input params - benchmark: {},  message: {}'.format(self._name, str(e)))
```

Comment:
Set return code for this error:
`self._result.set_return_code(ReturnCode.INVALID_ARGUMENT)`, otherwise the return code will be SUCCESS.
Please also add it for cublas_function benchmark.

Reply:
added, please check